### ISSUE TITLE
didn't compatible with the github.com/redis/go-redis@v9.2.0

### ISSUE THREAD

**Issue by KaptinLin** [ThreeDotsLabs/watermill-redisstream#29]
get errors below:
```
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\publisher.go:93:23: p.client.XAdd undefined (type redis.UniversalClient has no field or method XAdd)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:203:25: s.client.XGroupCreateMkStream undefined (type redis.UniversalClient has no field or method XGroupCreateMkStream)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:297:25: s.client.XReadGroup undefined (type redis.UniversalClient has no field or method XReadGroup)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:307:25: s.client.XRead undefined (type redis.UniversalClient has no field or method XRead)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:372:23: s.client.XPendingExt undefined (type redis.UniversalClient has no field or method XPendingExt)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:396:24: s.client.XClaim undefined (type redis.UniversalClient has no field or method XClaim)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:448:25: s.client.XInfoConsumers undefined (type redis.UniversalClient has no field or method XInfoConsumers)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:461:23: s.client.XGroupDelConsumer undefined (type redis.UniversalClient has no field or method XGroupDelConsumer)
\github.com\!three!dots!labs\watermill-redisstream@v1.2.1\pkg\redisstream\subscriber.go:558:7: p.XAck undefined (type redis.Pipeliner has no field or method XAck)
```

**Comment by minghsu0107** [comment]
@KaptinLin I think the API for `redis.UniversalClient` is broken after they refactor `go-redis`

**Comment by minghsu0107** [comment]
@KaptinLin Someone has already created a PR addressing this issue
FYR: https://github.com/redis/go-redis/pull/2725

**Comment by minghsu0107** [comment]
Fixed in v9.2.1: https://github.com/redis/go-redis/releases/tag/v9.2.1
